docs/guide/apm-release-notes.asciidoc

```diff
@@ -17,15 +17,40 @@ For a full list of changes, see the
 * <<release-notes-6.4.1>>
 * <<release-notes-6.4.0>>
 
+//NOTE: The notable-highlights tagged regions are re-used in the
+//Installation and Upgrade Guide
+// tag::notable-v8-highlights[]
+
+// end::notable-v8-highlights[]
+
 [[release-highlights-7.2.0]]
 === APM version 7.2.0
 
 [float]
 ==== New features
 
+// tag::notable-v72-highlights[]
 *APM Server*
 
-* Support for {apm-server-ref-v}/ilm.html[index lifecycle management (ILM)]
+* Added support for {apm-server-ref-v}/ilm.html[index lifecycle management (ILM)]:
+ILM automates the management of your indices over time,
+by rolling over data to new indices when the existing indices reach the specified size or age.
+* Added Geo-IP processing to the default ingest pipeline:
+The Geo-IP pipeline takes an extracted IP address from RUM events and stores it in the `client.geo` field.
+This makes it much easier to use location data in Kibana's Visualize maps and Maps app directly.
+Pipelines must be <<register.ingest.pipeline.enabled,enabled>>.
```

Comment:
This link fails when this content is re-used in the Installation and Upgrade Guide.  Can you change it to an external-friendly link?